### DISCUSSION TITLE
Simple DEMO of rendering GLFW window inside imgui window

### DISCUSSION THREAD

**Discussion by Yuchenhenry** [ocornut/imgui#4695] [New Users Build/Link/Run issues ONLY!]
Hello! I am a new learner on the Dear ImGUI. Recently I am working on a simple application with GLFW. Trying to see if there is a simple DEMO of how to render the GLFW window inside the ImGUI window. I want to have a scene window to present the model while a control window for detailed properties for GLFW functions. Any thoughts are welcome! 

Thank you very much!

**Comment by ocornut** [comment]
You cannot render a "GLFW window" inside another thing, it's not a meaningful question or statement as GLFW deals with native/platform windows (e.g. MS-Windows windows).

What you maybe mean is trying to render the output of your OpenGL rendering in a Dear ImGui window.
For this you will need to render your OpenGL scene into a texture. There are many OpenGL tutorials to do that and it has nothing to do with GLFW nor with Dear ImGui, e.g.
https://www.google.com/search?q=opengl+render+to+texture

Once you have an OpenGL texture with your scene you can render it in an Dear ImGui window with `ImGui::Image()` just like you would with a static image: https://github.com/ocornut/imgui/wiki/Image-Loading-and-Displaying-Examples
